packages/babel-plugin-transform-class-properties/test/fixtures/private-loose/assignment/output.js

```diff
@@ -12,10 +12,10 @@ var Foo = /*#__PURE__*/function () {
   babelHelpers.createClass(Foo, [{
     key: "test",
     value: function test(other) {
-      babelHelpers.classPrivateFieldLooseBase(this, _foo)[_foo] += 1;
-      babelHelpers.classPrivateFieldLooseBase(this, _foo)[_foo] = 2;
-      babelHelpers.classPrivateFieldLooseBase(other.obj, _foo)[_foo] += 1;
-      babelHelpers.classPrivateFieldLooseBase(other.obj, _foo)[_foo] = 2;
+      babelHelpers.classPrivateFieldGetLoose(this, _foo, 1)[_foo] += 1;
```

Comment:
Isn't this a size increase in general? It looks shorter because the helepr name is shorter, but once you minify the helper name to 1 char this has `,1` more.